doc/decisions/decision_process.md

```diff
@@ -0,0 +1,137 @@
+# Decision Process
+
+## Problem
+
+Simply discussing in an issue and then doing an implementation is okay for non-substantial changes.
+But substantial decisions must be made transparently and in a participative way.
+
+## Constraints
+
+- All relevant information about decisions must be within Elektra's repository.
+- All decisions must go through at least two review rounds, with a merge in between.
+- At least two people need to be in favor of the decision in both rounds.
+- [Documentation guidelines](/doc/contrib/documentation.md) apply.
+- During the decision process, the PRs constantly get updated:
+  - Make changes as new commits to the pull request.
+  - Questions in the PRs are answered by updating the PR and answering with the commit id.
+  - As generally recommended in Elektra, do not squash after they are visible on the pull request.
+  - Rebase only if the decision was already accepted and has a merge conflict.
+- For reviewers:
+  - Prefer to directly give suggestions how to change sentences.
+  - General points should be asked in the root of "Conversation" and not at random sentences in the review.
+
+## Assumptions
+
+- People want to be informed about or even participate in what Elektra looks like in the future.
+- We will always be able to reach an consensus.
+  We don't need a vote or a benevolent dictatorship.
+- Unlike the Rust Decision process, decisions in Elektra do not have a disadvantage if they were flawed in early stages.
+  Only the end results counts.
+
+## Considered Alternatives
+
+- Issues like https://issues.libelektra.org/4521
+- PEPs: https://peps.python.org
+- RFCs: https://www.ietf.org/standards/rfcs/
+- Change requests: https://en.wikipedia.org/wiki/Change_request
+
+## Decision
+
+Decisions need to:
+
+- be implementable within the next major release
+- be according to [Elektra's goals](/doc/GOALS.md)
+- first being decided upon using the decision process described here
+
+We base our decision process and template on:
+
+- [''using patterns to capture architectural decisions''](https://dl.acm.org/doi/10.1109/MS.2007.124),
```

Comment:
I'm not sure if basing the process onto a paper behind a paywall is such a good idead for an open-source project. Is there any way contributors outside of an academic network can access this?

Reply:
Seems like Vienna University hosts it directly, not sure about the legality though: http://eprints.cs.univie.ac.at/2345/1/02_Using_Patterns_to_Capture.pdf